src/ui/public/vislib_vis_type/vislib_renderbot.js

```diff
@@ -46,6 +46,7 @@ module.exports = function VislibRenderbotFactory(Private) {
   VislibRenderbot.prototype.buildChartData = buildChartData;
   VislibRenderbot.prototype.render = function (esResponse) {
     this.chartData = this.buildChartData(esResponse);
+    this.chartData.title = this.vis.title || '';
```

Comment:
This in my experience has never been undefined, it's usually 'New Visualization' when I would expect that, put the OR for safekeeping.
Added this when i thought the filters were panel specific and started implementing this for that, found out they weren't and left this piece for the future, and since it shouldn't be undefined anyway